AgileCoffeeShop/Coordinate.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace AgileCoffeeShop
+{
+    public class Coordinate
+    {
+        private double _xCoordinate;
```

Comment:
Why did you decided to use this naming convention?

Reply:
I decided to use this convention because it is a C# convention that I'm used to.

Reply:
And also because I started to read the Clean Code book.